source/Octopus.Tentacle.Tests.Integration/Support/TentacleTestCaseSourceAttribute.cs

```diff
@@ -164,12 +165,13 @@ private IEnumerable<ITestCaseData> GetTestCasesFor(IMethodInfo method)
                             parms = new TestCaseParameters(args);
                         }
 
-                        // If the test is trying to test .net6.0 TentacleClient against .net48 Tentacle,
-                        // then add the 'Net60ClientNet48Tentacle' category so we can potentially run these tests in a different test run
+                        // If the test is trying to test .net8.0 TentacleClient against .net48 Tentacle,
+                        // then add the 'Net80ClientNet48Tentacle' category so we can potentially run these tests in a different test run
                         #if !NETFRAMEWORK
                         if (item is TentacleConfigurationTestCase {TentacleRuntime: TentacleRuntime.Framework48} testCase)
                         {
                             parms.Properties.Add(PropertyNames.Category, Net60ClientNet48Service);
```

Comment:
Can we remove this line already?

Reply:
Not quite yet. Old build configurations still reference the net6 category. Once this has merged, the build configuration can switch to the net8 category and then we can remove it.